Build.PL

```diff
@@ -211,6 +211,7 @@ my $builder = $class->new(
                 'MooseX::StrictConstructor'       => '0.08',
                 'namespace::autoclean'            => '0.09',
                 'Net::LDAP'                       => 0,
+                'open'                            => 0,
```

Comment:
On 15/07/2016 09:41, dkj wrote:
> > +                'open'                            => 0,
>
> Is this required? i.e. is it not in core Perl?
>
It is core Perl. However, modules go in and out of core occasionally. 
Also, sometimes (not this case, I admit), one has to bump the version 
that is in core. And, I guess, it matters if one installs dependencies 
with an option to have latest even if a satisfactory version is already 
available. But ready to remove it if you think it would be better this way.


-- 
 The Wellcome Trust Sanger Institute is operated by Genome Research 
 Limited, a charity registered in England with number 1021457 and a 
 company registered in England with number 2742969, whose registered 
 office is 215 Euston Road, London, NW1 2BE.